lib/web_ui/dev/utils.dart

```diff
@@ -224,8 +224,7 @@ class ProcessManager {
     return (await eval()).stderr;
   }
 
-  @alwaysThrows
-  void _throwProcessException({required String description, int? exitCode}) {
+  Never _throwProcessException({required String description, int? exitCode}) {
```

Comment:
You may be able to remove the `package:meta` import too? 🤔

Reply:
No they use `@immutable`